mslice/plotting/pyplot.py

```diff
@@ -2603,7 +2624,7 @@ def loglog(*args, **kwargs):
 
 # This function was autogenerated by boilerplate.py.  Do not edit as
 # changes will be lost
-@_autogen_docstring(Axes.magnitude_spectrum)
+#@_autogen_docstring(Axes.magnitude_spectrum)
```

Comment:
Ok. So it turned out I just need to put the whole of those blocks in an `if` - don't need to do anything fancy with conditional decorators.